src/Psl/Dict/drop.php

```diff
@@ -9,8 +9,8 @@
  *
  * Examples:
  *
- *      Dict\drop([1, 2, 3, 4, 5], 3)
- *      => Dict(4, 5)
+ *      Dict\drop(['a' => 1, 'b' => 2, 'c' => 3, 'd' => 4], 2)
+ *      => Dict('c' => 3, 'd' => 4)
```

Comment:
I've used the old examples from `Dict\drop()` and `Dict\slice()` in their `Vec` counterparts, I think they make more sense there.